docs/usage/configuration.rst

```diff
@@ -12,7 +12,7 @@ Seeding Randomness
     :caption: Seeding the factory's 'random.Random'
     :language: python
 
-Seeding randomness allows you to control the random generation of values produced by the factory. This affects all 'random.Random'
+Seeding randomness allows you to control the random generation of values produced by the factory. This affects all ``random.Random``
```

Comment:
```suggestion
Seeding randomness allows you to control the random generation of values produced by the factory. This affects all `:class:`:random.Random`
```

Good with this as it is, but it might be better to just link to this with something like this

Reply:
Cheers! I'll try :)